api/tests/routes/native/v1/offers_test.py

```diff
@@ -278,6 +278,47 @@ def test_get_thing_offer(self, client):
         assert response.json["venue"]["isPermanent"]
         assert response.json["stocks"][0]["features"] == []
 
+    @pytest.mark.parametrize(
+        "provider_class,ff_name,ff_value,expected",
+        [
+            ("EMSStocks", "DISABLE_EMS_EXTERNAL_BOOKINGS", True, True),
+            ("EMSStocks", "DISABLE_EMS_EXTERNAL_BOOKINGS", False, False),
+            ("CGRStocks", "DISABLE_CGR_EXTERNAL_BOOKINGS", True, True),
+            ("CGRStocks", "DISABLE_CGR_EXTERNAL_BOOKINGS", False, False),
+            ("CDSStocks", "DISABLE_CDS_EXTERNAL_BOOKINGS", True, True),
+            ("CDSStocks", "DISABLE_CDS_EXTERNAL_BOOKINGS", False, False),
+            ("BoostStocks", "DISABLE_BOOST_EXTERNAL_BOOKINGS", True, True),
+            ("BoostStocks", "DISABLE_BOOST_EXTERNAL_BOOKINGS", False, False),
+            ("BoostStocks", "DISABLE_EMS_EXTERNAL_BOOKINGS", True, False),
+        ],
+    )
+    def test_get_sync_event_offer(self, client, provider_class, ff_name, ff_value, expected):
+        provider = get_provider_by_local_class(provider_class)
+        product = offers_factories.ProductFactory(thumbCount=1, subcategoryId=subcategories.SEANCE_CINE.id)
+        offer = offers_factories.OfferFactory(
+            product=product,
+            venue__isPermanent=True,
+            subcategoryId=subcategories.SEANCE_CINE.id,
+            lastProvider=provider,
+        )
+        offers_factories.EventStockFactory(offer=offer, price=12.34)
+        providers_factories.VenueProviderFactory(venue=offer.venue, provider=provider)
+
+        offer_id = offer.id
+        with override_features(**{ff_name: ff_value}):
+            with assert_no_duplicated_queries():
+                response = client.get(f"/native/v1/offer/{offer_id}")
+
+        assert response.status_code == 200
+        assert response.json["stocks"][0]["price"] == 1234
+        assert response.json["stocks"][0]["remainingQuantity"] == 1000
+        assert response.json["subcategoryId"] == "SEANCE_CINE"
+        assert response.json["isEducational"] is False
+        assert not response.json["isExpired"]
+        assert response.json["venue"]["isPermanent"]
+        assert response.json["stocks"][0]["features"] == []
+        assert response.json["isExternalBookingsDisabled"] is expected
```

Comment:
```suggestion
        assert response.status_code == 200
        assert response.json["isExternalBookingsDisabled"]
```
je ne vois pas le lien entre les autres `assert` et le test, tu peux aussi supprimer l'appel à `EventStockFactory`